scripts/docs/parse-arguments.mjs

```diff
@@ -5,51 +5,87 @@
 
 /* eslint-env node */
 
-import minimist from 'minimist';
+import { parseArgs } from 'util';
 import replaceKebabCaseWithCamelCase from '../utils/replacekebabcasewithcamelcase.mjs';
 
 /**
- * @param {Array.<String>} args An array containing modifiers for the executed command.
+ * @param {Array<string>} args An array containing modifiers for the executed command.
  * @return {DocumentationOptions}
  */
 export default function parseArguments( args ) {
-	const options = minimist( args, {
-		boolean: [
-			'skip-api',
-			'skip-snippets',
-			'skip-validation',
-			'skip-guides',
-			'skip-ckeditor5',
-			'skip-commercial',
-			'production',
-			'watch',
-			'verbose'
-		],
-		string: [
-			'snippets',
-			'guides'
-		],
-		default: {
-			'skip-api': false,
-			'skip-snippets': false,
-			'skip-validation': false,
-			'skip-guides': false,
-			'skip-ckeditor5': false,
-			'skip-commercial': false,
-			production: false,
-			watch: false,
-			verbose: false,
-			snippets: [],
-			guides: []
+	const { values } = parseArgs( {
+		args,
+		strict: true,
```

Comment:
Now the arguments will be validated. Passing unknown arguments will result in an error.